runtime-modules/proposals/codex/src/weights.rs

```diff
@@ -338,14 +339,12 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ProposalDiscussion ThreadById (r:0 w:1)
 	// Storage: ProposalEngine Proposals (r:0 w:1)
 	// Storage: ProposalEngine DispatchableCallCode (r:0 w:1)
-	fn create_proposal_amend_constitution(i: u32, t: u32, d: u32, ) -> Weight {
-		(112_251_000 as Weight)
-			// Standard Error: 9_000
-			.saturating_add((1_445_000 as Weight).saturating_mul(i as Weight))
-			// Standard Error: 52_000
-			.saturating_add((492_000 as Weight).saturating_mul(t as Weight))
-			// Standard Error: 52_000
-			.saturating_add((594_000 as Weight).saturating_mul(d as Weight))
+	fn create_proposal_amend_constitution(i: u32, t: u32, _d: u32, ) -> Weight {
+		(121_461_000 as Weight)
+			// Standard Error: 8_000
+			.saturating_add((1_323_000 as Weight).saturating_mul(i as Weight))
+			// Standard Error: 66_000
+			.saturating_add((632_000 as Weight).saturating_mul(t as Weight))
 			.saturating_add(T::DbWeight::get().reads(7 as Weight))
```

Comment:
The dependency on `d` was removed. Probably it was just some interference during weight function generation.